Source/WebCore/html/HTMLImageElement.cpp

```diff
@@ -438,6 +442,10 @@ void HTMLImageElement::attributeChanged(const QualifiedName& name, const AtomStr
     if (isImageMenuEnabled())
         ImageControlsMac::updateImageControls(*this);
 #endif
+
+#if ENABLE(SPATIAL_IMAGE_CONTROLS)
+    SpatialImageControls::updateSpatialImageControls(*this);
```

Comment:
This is not an appropriate place to do this. This function runs every time an attribute is mutated on an image element.

Reply:
Please add it under `AttributeNames::controlsAttr`.